Little fixes

Snowstar38 opened the pull request:
## About The Pull Request

A few small fixes:

- Use shorten_text_to_fit to dynamically truncate names in patrol screen (this was accidentally missed when I originally updated all shortened text to use shorten_text_to_fit last year)
- Polishes up some text for a new cat patrol
- Add living status constraint for all thoughts with r_c and no living status, fix indentation in thought .jsons
- Fixes other Clan name appearing as "OtherClanClan" in history text

Snowstar38 committed:
Shorten text to fit, fix to patrol text

some fixes

Snowstar38 committed:
Merge branch 'ClanGenOfficial:development' into Little-fixes

Snowstar38 committed:
gen_death_badger_anylead2 r_c fix

scribblecrumb commented on resources/dicts/events/death/general.json:
```diff
@@ -3916,7 +3916,7 @@
         },
         "history": [{
-            "cats": ["m_c" ],
+            "cats": [ "m_c", "r_c"],
```
The "cats" list is meant for specifying which cats can have the histories in this block assigned to them.  As r_c doesn't die in this event, I'm fairly certain that, at best, this has no effect (at worst, might break something?)

Snowstar38 committed:
Merge branch 'ClanGenOfficial:development' into Little-fixes

Snowstar38 committed:
Add living status constraint for all thoughts with r_c and no living status

I had my leader have a "cold" personality thought about their dead StarClan guide and realized that a lack of random_living_status was the issue. I also fixed the indentation of some of the json files.

Snowstar38 committed:
Update rogue.json

Snowstar38 committed:
Update PatrolScreen.py

Ryos00 reviewed (commented):
gonna let you add the changes scribble requested before approving, everything else looks fine to me

Snowstar38 committed:
Fix "OtherClanClan" in history tab

Snowstar38 committed:
Merge branch 'ClanGenOfficial:development' into Little-fixes

Snowstar38 committed:
Merge branch 'Little-fixes' of https://github.com/Snowstar38/clangen into Little-fixes

Snowstar38 committed:
Revert "gen_death_badger_anylead2 r_c fix"

This reverts commit cc6b9c3418245afe3f296f7a44b612e181aa7f37.

Snowstar38 commented:
@scribblecrumb I removed that bit for you :)
